src/sharedHelpers/selectImagesFromGallery.js

```diff
@@ -0,0 +1,84 @@
+import { Platform } from "react-native";
+import ImagePicker from "react-native-image-crop-picker";
+
+const handleIosImage = image => {
```

Comment:
Hmm, could explain more on the reliable/easy part?

`react-native-image-crop-picker` pulls the exif data natively, but there is no normalization layer, so the shape/attributes for ios/android simply come back differently. This js code just normalizes data  so the app can use it effectively.

I ask this because `native-exif-reader` isn't able to use the files from `react-native-image-crop-picker`, not exactly sure why

Reply:
@kueda, any thoughts on this? I know Yaron spent a while making sure we can get exif data natively, but I'm not 100% certain why we need that or if that's a dealbreaker here for using `react-native-image-crop-picker`.

Reply:
@budowski did this work as a part of #144. I believe we wrote our own EXIF library because we were unable to consistently extract `GPSHPositioningError` using existing react native libraries. @Chrischuck, [this image](https://user-images.githubusercontent.com/22691/187563722-4d2b4a4a-a9f4-45df-9ba4-595e36528e5a.jpg) should have that tag. Are you seeing it getting extracted with your code in this branch on *both* iOS and Android?

Reply:
@kueda looks like it isn't able to get that tag. If I recall, `react-native-image-picker` worked fine w/ exif reader but the resizer didn't like it. Might be easier to go that route and just find a different resizer library